caas/kubernetes/provider/utils/annotations_test.go

```diff
@@ -17,30 +18,30 @@ type annotationSuite struct {
+	c.Assert(utils.AnnotationVersionKey(constants.LabelVersion1), gc.DeepEquals, "juju.is/version")
 
-	c.Assert(utils.AnnotationModelUUIDKey(true), gc.DeepEquals, "juju-model")
-	c.Assert(utils.AnnotationModelUUIDKey(false), gc.DeepEquals, "model.juju.is/id")
+	c.Assert(utils.AnnotationModelUUIDKey(constants.LegacyLabelVersion), gc.DeepEquals, "juju-model")
+	c.Assert(utils.AnnotationModelUUIDKey(constants.LabelVersion1), gc.DeepEquals, "model.juju.is/id")
 
-	c.Assert(utils.AnnotationControllerUUIDKey(true), gc.DeepEquals, "juju.io/controller")
-	c.Assert(utils.AnnotationControllerUUIDKey(false), gc.DeepEquals, "controller.juju.is/id")
+	c.Assert(utils.AnnotationControllerUUIDKey(constants.LegacyLabelVersion), gc.DeepEquals, "juju.io/controller")
+	c.Assert(utils.AnnotationControllerUUIDKey(constants.LabelVersion1), gc.DeepEquals, "controller.juju.is/id")
 
-	c.Assert(utils.AnnotationControllerIsControllerKey(true), gc.DeepEquals, "juju.io/is-controller")
-	c.Assert(utils.AnnotationControllerIsControllerKey(false), gc.DeepEquals, "controller.juju.is/is-controller")
+	c.Assert(utils.AnnotationControllerIsControllerKey(constants.LegacyLabelVersion), gc.DeepEquals, "juju.io/is-controller")
+	c.Assert(utils.AnnotationControllerIsControllerKey(constants.LabelVersion1), gc.DeepEquals, "controller.juju.is/is-controller")
 
-	c.Assert(utils.AnnotationUnitKey(true), gc.DeepEquals, "juju.io/unit")
-	c.Assert(utils.AnnotationUnitKey(false), gc.DeepEquals, "unit.juju.is/id")
+	c.Assert(utils.AnnotationUnitKey(constants.LegacyLabelVersion), gc.DeepEquals, "juju.io/unit")
+	c.Assert(utils.AnnotationUnitKey(constants.LabelVersion1), gc.DeepEquals, "unit.juju.is/id")
 
-	c.Assert(utils.AnnotationCharmModifiedVersionKey(true), gc.DeepEquals, "juju.io/charm-modified-version")
-	c.Assert(utils.AnnotationCharmModifiedVersionKey(false), gc.DeepEquals, "charm.juju.is/modified-version")
+	c.Assert(utils.AnnotationCharmModifiedVersionKey(constants.LegacyLabelVersion), gc.DeepEquals, "juju.io/charm-modified-version")
+	c.Assert(utils.AnnotationCharmModifiedVersionKey(constants.LabelVersion1), gc.DeepEquals, "charm.juju.is/modified-version")
 
-	c.Assert(utils.AnnotationDisableNameKey(true), gc.DeepEquals, "juju.io/disable-name-prefix")
-	c.Assert(utils.AnnotationDisableNameKey(false), gc.DeepEquals, "model.juju.is/disable-prefix")
+	c.Assert(utils.AnnotationDisableNameKey(constants.LegacyLabelVersion), gc.DeepEquals, "juju.io/disable-name-prefix")
+	c.Assert(utils.AnnotationDisableNameKey(constants.LabelVersion1), gc.DeepEquals, "model.juju.is/disable-prefix")
 
-	c.Assert(utils.AnnotationKeyApplicationUUID(true), gc.DeepEquals, "juju-app-uuid")
-	c.Assert(utils.AnnotationKeyApplicationUUID(false), gc.DeepEquals, "app.juju.is/uuid")
+	c.Assert(utils.AnnotationKeyApplicationUUID(constants.LegacyLabelVersion), gc.DeepEquals, "juju-app-uuid")
+	c.Assert(utils.AnnotationKeyApplicationUUID(constants.LabelVersion1), gc.DeepEquals, "app.juju.is/uuid")
```

Comment:
We need tests for LabelVersion2 to prove it works like LabelVersion1
Maybe a for loop for anything other than LabelVersion0